### PR TITLE
Overlay break solution

### PR DESCRIPTION
**Pull request by GuiiMoraes**
I am trying to help with breaking overlay with multiple icon clicks.
This will deactivate the button and will not accept multiple clicks while setTimeOut is awaited.
I changed a button that was a div tag to a tag button to accept attribute deactivation.
Added and removed this disable attribute while waiting for setTimeOut
Border removed for this button tag
Removed package.json to use yarn.lock to manage all dependencies.

### CODE REVIEW

File: src/darkmode.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ export default class Darkmode {
         height: 3rem;
         position: fixed;
         border-radius: 50%;
+        border:none;
```
Comment (MrSkwiggs) [inline src/darkmode.js]:
```suggestion
        border: none;
```

**Comment by MrSkwiggs** [comment]
@sandoche give this some love :)

**Comment by sandoche** [comment]
@GuiiMoraes thank, I need to test and to put back: package-lock.json and so on.

**Comment by sandoche** [comment]
I merged it manually, after fixing back the lock files, thanks a lot!

**Comment by MrSkwiggs** [comment]
A christmas miracle 😄

**Pull request closed without merging**